File: pgm/ABC_Visual_Basic/ABC_164_A.vb
' Imports System.Collections.Generic
' Imports System.Text.RegularExpressions

Class Main
    Public Shared Sub Main()
        Dim S, W As Integer
        Dim arr() As String
        arr = Split(Console.ReadLine())
        S = arr(0) : W = arr(1)
        Console.WriteLine(If(S <= W, "unsafe", "safe"))
        Return
    End Sub
End Class

File: pgm/ABC_C#/ABC_164_A.cs
using System;
// using System.Collections.Generic;
// using System.Linq;

class Program
{
    static void Main()
    {
        String[] arr = Console.ReadLine().Split();
        int S, W;
        S = int.Parse(arr[0]);
        W = int.Parse(arr[1]);
        Console.WriteLine(S <= W ? "unsafe" : "safe");
        return;
    }
}